ipatests/test_integration/test_trust.py

```diff
@@ -95,6 +90,11 @@ def remove_trust(self, ad):
 
 class TestTrust(BaseTestTrust):
 
+    dn = DN(
```

Comment:
1. The variable name tells nothing about its contents.
2. It does not need to be class member as it is used in a single test case